HW17/README.md

```diff
@@ -0,0 +1,29 @@
+# HW17
+
+2.
+![1](/HW17/images/02-1.png)
+![2](/HW17/images/02-2.png)
+
+3.
+![3](/HW17/images/03-1.png)
+![4](/HW17/images/03-2.png)
+
+4.
+![5](/HW17/images/04.png)
+
+5.
+![6](/HW17/images/05.png)
+
+6.
+![7](/HW17/images/06-1.png)
+![8](/HW17/images/06-2.png)
+![9](/HW17/images/06-3.png)
+
+7.
+![10](/HW17/images/07.png)
```

Comment:
This is not quit right.
https://medium.com/@jacob.swanson.n/making-docker-use-different-ip-ranges-on-ubuntu-27ba6cbb825d